app/experimenter/experiments/constants/nimbus.py

```diff
@@ -16,21 +16,15 @@ class NimbusTargetingConfig:
     name="All English users",
     slug="all_english",
     description="All users in en-* locales using the release channel.",
-    targeting=(
-        "localeLanguageCode == 'en' && browserSettings.update.channel "
-        "== '{firefox_channel}'"
-    ),
+    targeting="localeLanguageCode == 'en'",
     desktop_telemetry="STARTS_WITH(environment.settings.locale, 'en')",
 )
 
 TARGETING_US_ONLY = NimbusTargetingConfig(
     name="US users (en)",
     slug="us_only",
     description="All users in the US with an en-* locale using the release channel.",
-    targeting=(
-        "localeLanguageCode == 'en' && region == 'US' && "
-        "browserSettings.update.channel == '{firefox_channel}'"
-    ),
+    targeting="localeLanguageCode == 'en' && region == 'US'",
```

Comment:
Definitely worth considering whether we want to know historically whether an experiment in the past had a different targeting config for the same slug. Maybe @k88hudson would have some insight on if we want to track evolution of targeting config.

Reply:
Yep totally worth discussing, but also probably outside the scope of this PR

Reply:
Yup.